samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py

```diff
@@ -80,18 +90,10 @@ def training_input(input_name, s3_uri):
 def mnist_classification(
     region="us-east-1",
     # General component inputs
-    output_encryption_key="",
     instance_type="ml.m5.2xlarge",
```

Comment:
This

Reply:
retaining this, since it depends on account